java/java.completion/nbproject/project.properties

```diff
@@ -31,3 +31,6 @@ test.timeout=1200000
 
 # requires nb.javac for compiling of tests on Mac
 requires.nb.javac=true
+
+test.jms.flags=--limit-modules=java.base,java.logging,java.xml,java.prefs,java.desktop,java.management,java.instrument,jdk.zipfs,java.scripting,java.naming
+test.bootclasspath.prepend.args=-Dno.netbeans.bootclasspath.prepend.needed=true
```

Comment:
this has almost the same effect as the old CI config line:
```
OPTS_11=-Dtest.run.args=--limit-modules=java.base,java.logging,java.xml,java.prefs,java.desktop,java.management,java.instrument,jdk.zipfs,java.scripting,java.naming -Dtest.bootclasspath.prepend.args=-Dno.netbeans.bootclasspath.prepend.needed=true
```
the difference is that the entire run args property isn't overwritten which is set up here:
https://github.com/apache/netbeans/blob/30a8cefa5a73f258b61939729e845672e66826fe/nbbuild/templates/projectized.xml#L293


this has however the side effect of warnings like:
`WARNING: Unknown module: jdk.compiler specified to --add-exports`

I am wondering if i should set `metabuild.jms-flags.jvm` to an empty string in the project configs?

Reply:
actually, i got an idea while waiting for Santa.

we can simply unset the default flags of the compiler category:
```
# remove default compiler JMS flags so that we don't get "Unknown module" warnings due to --limit-modules
jms-compiler.flags.jvm=
```